src/main.py

```diff
@@ -0,0 +1,44 @@
+from collection import HTTPPacketCollector
+from statistics_management import StatisticsManager
+from alerting import AlertManager
+from argparse import ArgumentParser
+from time import sleep
+import utils
+
+logger = utils.get_logger(__name__)
+
+def main(args):
+    interval_count = 0
+    statistics_check_interval_seconds = 10
+    alert_check_interval = 12
+    logger.info(f'Starting program and initializing packet collector.')
+    logger.debug(f'Starting program with check interval: {statistics_check_interval_seconds} and alert interval:'
+                 f'{alert_check_interval}.')
+    packet_collector = HTTPPacketCollector()
+    packet_collector.start_packet_collection(args.interface if args.interface else None)
+    alerts = AlertManager(args.alert_trigger_threshold)
+    statistics = StatisticsManager(alert_check_interval)
+
+    statistics.show_all_statistics()
+
+    while True:
+        sleep(statistics_check_interval_seconds)
+        interval_count += 1
```

Comment:
Why not sleep for 12 seconds? Not sure what `interval_count` is used for besides being 0 or 1.

Probably easier to talk offline about this.